gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-environments.yaml

```
@@ -1871,6 +1905,68 @@ components:
           FOLDER: "#/components/schemas/CreatePortalNavigationFolder"
           LINK: "#/components/schemas/CreatePortalNavigationLink"
 
+    BaseUpdatePortalNavigationItem:
+      type: object
+      description: Base portal navigation item for Update
+      properties:
+        type:
+          $ref: "#/components/schemas/PortalNavigationItemType"
+        title:
+          type: string
+          description: The title of the navigation item
+        order:
+          type: integer
+          minimum: 0
+          description: The order of the navigation item, from 0 to MAX + 1, where MAX is the maximum existing order within the target parent. If not provided or greater than MAX, the new item is appended to the end of the target parent.
```

I missed this on the last review
```suggestion
          description: The order of the navigation item, from 0 to MAX, where MAX is the maximum existing order within the target parent. If greater than MAX, the item moved to the end of the target parent.